app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -227,7 +227,23 @@ class BrowserToolbarView(
 
     fun expand() {
         if (!settings.shouldUseBottomToolbar) {
-            layout.app_bar?.setExpanded(true)
+            view.app_bar?.setExpanded(true)
+        }
+    }
+
+    /**
+     * Performs all actions that refer to the BrowserToolbarView that don't require user action.
```

Comment:
nit: Hm can we rephrase this? I didn't really understand the "that don't require user action" part, but maybe I am missing something...

Maybe something along these lines:

Triggers async inflation of this [BrowserToolbarView] and invokes the provided [action] once inflation is completed so dependent functionality (views, features) can be initialized.